src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -7,20 +7,23 @@
         <label class="font-weight-bold pl-2">{{ orgPersonLabel }}</label>
       </div>
 
-      <!-- Instructional people and roles text (BEN corrections only)-->
-      <article v-if="isBenCorrectionFiling" class="section-container">
+      <!-- Instructional people and roles text (base corrections only)-->
+      <article v-if="isBaseCorrectionFiling" class="section-container">
         This application must include the following:
         <ul>
           <li>
-            <v-icon v-if="dirValid" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else color="red" class="dir-invalid">mdi-close</v-icon>
-            <span class="ml-2">At least one Director</span>
+            <v-icon v-if="haveMinimumDirectors" color="green darken-2" class="dir-valid">mdi-check</v-icon>
+            <template v-else>
+              <v-icon color="red" class="dir-invalid">mdi-close</v-icon>
+              <span v-if="isTypeBC || isTypeBEN || isTypeULC" class="ml-2">At least one Director</span>
+              <span v-if="isTypeCCC" class="ml-2">At least three Directors</span>
+            </template>
```

Comment:
The rules and text should be in resource files -- future work.

See also line 214.